index.html

```diff
@@ -88,7 +88,7 @@ <h2>Who?</h2>
             <li><a title="PowerSync" href="https://github.com/powersync-ja/powersync-service" ><img src="./logos/powersync.svg" alt="PowerSync logo" /></a></li>
             <li><a title="Sentry" href="https://github.com/getsentry/self-hosted" ><img src="./logos/sentry.svg" alt="Sentry logo" /></a></li>
             <li><a title="Vyuh Framework" href="https://github.com/vyuh-tech/vyuh" ><img src="./logos/vyuh.svg" alt="Vyuh logo" /></a></li>
-
+            <li><a title="Sweetr" href="https://github.com/sweetr-dev/sweetr.dev" ><img src="./logos/sweetr-dev.svg" alt="Sweetr logo" /></a></li>
```

Comment:
This is what is looks like after the update:
![image](https://github.com/user-attachments/assets/92e21dbe-f5d0-49a6-a82c-9774ecd6cda1)